macos/FluentUI/Button/Button.swift

```diff
@@ -263,6 +263,25 @@ open class Button: NSButton {
 		}
 	}
 
+	open override func viewDidChangeEffectiveAppearance() {
```

Comment:
I would move this to after the viewDidChangeBackingProperties() override to keep with the other "viewDid..." related methods.